data/systemd-user/snap-userd.service.in

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Snapd's user session agent
+Requires=snap-userd.socket
+
+[Service]
+# userd claims two D-Bus names on start.  Settings is the second one.
```

Comment:
Curious, does this mean that only one can be activated with a DBus call?

Reply:
No.  With `Type=dbus`, systemd treats the daemon as "starting" until the given bus name has been acquired on the bus.  As this daemon acquires two bus names, I'm just saying that we should wait until the second bus name is acquired for the daemon to be considered started.

Reply:
Since I split the session agent out from userd, it no longer has any D-Bus code.  So I switched it over to being `Type=notify`, which also got rid of the need for that comment.